static/babelrc.json

```diff
@@ -1,7 +1,12 @@
 {
-  "breakConfig": true,
   "sourceMap": "inline",
-  "blacklist": ["es6.forOf", "useStrict"],
-  "optional": ["asyncToGenerator"],
-  "stage": 0
+  "plugins": [
+    "add-module-exports",
+    "transform-async-to-generator",
+    "transform-class-properties",
+    "transform-es2015-modules-commonjs",
```

Comment:
Interesting, the only issue I could find was that the following code wasn't being generated:

```js
module.exports = exports['default']
```

This caused interoperability issue with e.g. CoffeeScript, but it was fixed by adding the `add-module-exports` transform (as recommended on https://babeljs.io/docs/plugins/transform-es2015-modules-commonjs/).